server/Config/default.json

```diff
@@ -0,0 +1,3 @@
+{
+  "mongoURI": "mongodb://127.0.0.1:27017/hackshare?gssapiServiceName=mongodb"
```

Comment:
Use `process.env` for URL.

Reply:
We should probably use dotenv unless 'config' has a way of importing env variables

Reply:
> We should probably use dotenv unless 'config' has a way of importing env variables

We have to export them manually, as did is `/config/db.js`
```
const mongoURI = config.get("mongoURI");
```



> Use `process.env` for URL.

The Database will be hosted on Cloud later which has specific IP, so let's leave it for now as it is.